ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils_filtered_tenants.py

```diff
@@ -126,6 +127,34 @@ def _filtered_vrfs(self, tenant: dict) -> list[dict]:
                     and l3_interface.get("interfaces") is not None
                 )
             ]
+
+            vrf["_l3_multicast_enabled"] = default(get(vrf, "l3_multicast.enabled"), get(tenant, "l3_multicast.enabled"))
+
+            if vrf["_l3_multicast_enabled"] is True and not get(self._hostvars, "evpn_multicast"):
```

Comment:
The check should be for `switch.evpn_multicast` since that includes a lot of protection and checks. so add a cached_property _evpn_multicast to avoid doing the deeper lookup again and again per vrf.